docs/src/components/componentList/drawerComponent/DrawerDocsComponent.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import Drawer from '../../../markdown/DrawerMarkdown.mdx';
```

Comment:
I didn't want the file name changed here. It's in a markdown folder and has a .mdx extension so we know its markdown. My issue was the import name as Drawer as this could be misconstrued as the component rather than the docs.

e.g.
import DrawerDocs from '../markdown/Drawer.mdx